Medicare and Medicaid/README.md

```diff
@@ -0,0 +1,37 @@
+# Medicare and Medicare Microdata
+
+Current Population Survey (CPS) has Medicare and Medicaid coverage information available at individual level but not benefit amounts. We imputed Medicare and Medicaid benefit from Medical Expenditure Panel Survey (MEPS) for each individual currently covered by either program. We used full year consolidated data file (HC171; 2014), obtained through [MEPS website](https://meps.ahrq.gov/data_stats/download_data_files_results.jsp?cboDataYear=All&cboDataTypeY=1%2CHousehold+Full+Year+File&buttonYearandDataType=Search&cboPufNumber=All&SearchTitle=Consolidated+Data).
+
+
+In the imputation process, we aimed at targeting total benefit expenditures, excluding administrative expenses, for non-institutionalized population. We obtain enrollee and benefit targets from [Medicaid Actuary Report](https://www.medicaid.gov/medicaid/finance/actuarial-report/index.html) and [Medicare Trustee's Report](https://www.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/ReportsTrustFunds/index.html)
+
+## Total number of recipients
+
+MEPS has already imputed coverage and benefit before distributing the datasets with their household and provider components in the survey. Thus, the number of recipients are fairly close to official reports for both programs. Medicare annual report claims it covered 53.8 million people in 2014, and MEPS Medicare recipients sum up to about 49.6 million, which is not far away since MEPS doesn�t cover institutionalized population. CPS Medicare section also has about 48.9 million recipients, so there�s not much need for imputation. But Medicaid has a slightly different story. Medicaid covers about 64 millions beneficiaries (point in time during FY 2014), while MEPS has about 74 million and CPS has about 54 million. MEPS seems to cover more people who enrolled for a short period of a year, so the total recipients are higher. But CPS is under-reported compared to Medicare. Again, if consider the population living inside institutions, CPS is not too far away. At this moment, we leave CPS coverage information as it is.
```

Comment:
> Medicare annual report claims it covered 53.8 million people in 2014, and MEPS Medicare recipients sum up to about 49.6 million, which is not far away since MEPS doesnÕt cover institutionalized population. CPS Medicare section also has about 48.9 million recipients, so thereÕs not much need for imputation.

Do we know the rough size of the institutionalized population? And does CPS include the institutionalized population?